ntc_templates/templates/index

```diff
@@ -796,6 +796,7 @@ juniper_junos_show_vlans.textfsm, .*, juniper_junos, sh[[ow]] vl[[ans]]
 juniper_screenos_get_route.textfsm, .*, juniper_screenos, get route
 
 linux_dmidecode_-t_processor.textfsm, .*, linux, dmidecode -t processor
+linux_nmcli_connection_show.textfsm, .*, linux, nmcli c[[onnections]] sh[[ow]]
```

Comment:
The `connection` keyword is singular, fixing that.

[referencing - Example 3.6.  Viewing all connections](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/7/html/networking_guide/sec-configuring_ip_networking_with_nmcli#sec-Brief_Selection_of_nmcli_Examples)

It's also possible to run a command as short as `nmcli c s`.

```suggestion
linux_nmcli_connection_show.textfsm, .*, linux, nmcli c[[onnection]] s[[how]]
```